src/Dfe.PlanTech.Web/Program.cs

```diff
@@ -34,14 +34,13 @@
 builder.Services.AddControllersWithViews();
 
 builder.Services.AddGovUkFrontend();
+var keyVaultUri = $"https://{builder.Configuration["KeyVaultName"]}.vault.azure.net/";
```

Comment:
Struggling to build the solution with this line in play - I've added the config to my appSettings, is there anything else required to get this working?

Reply:
Whoops. This was me enabling the keyvault locally for debugging; I didn't mean to check this in. I'll revert it. Speak to me about this though because it's good to know how to use locally for proper testing.